include/enmasse-kubernetes.jsonnet

```diff
@@ -10,7 +10,7 @@ local flavorConfig = import "flavor.jsonnet";
     "items": [ templateConfig.generate(use_sasl, with_kafka, false),
                addressController.deployment("enmasseproject/address-controller", "false", 0),
                restapiRoute.ingress(""),
-               addressController.service,
+               addressController.service("LoadBalancer"),
```

Comment:
I'm going to propose a new directory structure for the templates to cope better with 'addon' templates:

install/openshift/enmasse.json
install/openshift/enmasse-with-kafka.json
install/openshift/enmasse-with-sasldb.json

install/kubernetes/enmasse.json
install/kubernetes/addons/external-lb.json